### PR TITLE
Explain collapsable right sidebar that contains documentation 

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Explain collapsable right sidebar that contains documentation

### CODE REVIEW

**Comment by matks** [comment]
Could you please review this documentation @PrestaShop/prestashop-maintainers ❤️

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Back Office Help sidebar
+menuTitle: Back Office help sidebar
+---
+
+# The Back Office Help sidebar
+
+Back Office controller return Back Office HTML pages.
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:8]:
```suggestion
Back Office controller returns Back Office HTML pages.
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Most of these pages Back Office page have a common HTML structure:
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:10]:
```suggestion
Most of these Back Office pages have a common HTML structure:
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- a layout at the top
+- a menu navigation bar on the left
+- the page content at the center
+- a collapsable right sidebar that contains documentation 
+
+## About the help sidebar content
+
+When BO user clicks on 'Help' button, the sidebar opens and displays the related Documentation page for the Back Office page that the user browses.
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:18]:
```suggestion
When the BO user clicks on the 'Help' button, the sidebar opens and displays the related Documentation page for the Back Office page that the user browses.
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The content of this documentation page comes from help.prestashop.com which is a gateway that returns the content of doc.prestashop.com in the right format to be inserted into the Back Office.
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:20]:
```suggestion
This documentation page comes from help.prestashop.com which is a gateway that returns the content of doc.prestashop.com in the right format to be inserted into the Back Office.
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## How it is built for modern controllers
+
+The sidebar contains an HTML [object](https://www.w3schools.com/tags/tag_object.asp) (similar to iframe). When help.prestashop.com is fetched, the returned content is a HTML document fetched from help.prestashop.com by [`.load`](https://api.jquery.com/load/).
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:24]:
```suggestion
The sidebar contains an HTML [object](https://www.w3schools.com/tags/tag_object.asp) (similar to iframe). When help.prestashop.com is fetched, the returned content is an HTML document fetched from help.prestashop.com by [`.load`](https://api.jquery.com/load/).
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Fetch is performed with URLs structure:
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:26]:
```suggestion
Fetch is performed to URL with this structure:
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+`help.prestashop.com/<language code>/doc/<help context>?version=<version number>`
+
+| Element            | Description
+|--------------------|-----------------------------------------------------------------------
+| `<language code>`  | ISO 639-1 language code (2 letters, lower case)
+| `<help context>`   | The controller name for which help is requested
+| `<version number>` | Version number of current PrestaShop app calling for contextual help.
+
+Example : https://help.prestashop.com/fr/doc/AdminProducts?version=1.7.1.2
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:36]:
```suggestion
Example: https://help.prestashop.com/fr/doc/AdminProducts?version=1.7.1.2
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+## How it is built for legacy controllers
+
+The sidebar has an empty container. When help.prestashop.com is fetched, the returned content is a JS scripts that fills the empty container with content.
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:40]:
```suggestion
The sidebar has an empty container. When help.prestashop.com is fetched, the returned content is JavaScript that fills the empty container with content.
```

File: src/content/1.7/development/components/back-office-help-sidebar.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+Fetch is performed with URLs structure:
```
Comment (kpodemski) [inline src/content/1.7/development/components/back-office-help-sidebar.md:42]:
```suggestion
Fetch is performed to URL with this structure:
```

File: src/content/1.7/development/components/back-office-help-sidebar.md (hunk 2 above)
Comment (matks) [reply src/content/1.7/development/components/back-office-help-sidebar.md:10]: Bad writing 😞  I feel ashamed

Thank you for the proof reading

**Review by kpodemski — APPROVED** [review]

**Comment by matks** [comment]
Thanks for review

**Pull request merged**